src/main/scala/mesosphere/marathon/core/task/termination/impl/KillAction.scala

```diff
@@ -64,8 +64,8 @@ private[termination] object KillAction extends StrictLogging {
     val hasReservations = knownInstance.fold(false)(_.hasReservation)
 
     // TODO(PODS): align this with other Terminal/Unreachable/whatever extractors
-    val maybeCondition = knownInstance.map(_.state.condition)
-    val isUnkillable = maybeCondition.fold(false)(wontRespondToKill)
+    val maybeCondition = knownInstance.map(_.tasksMap.values.view.map(_.status.condition))
```

Comment:
What's the return type? Shouldn't we summarize the tasks condition into an instance condition.